src/VCreditCard.vue

```diff
@@ -115,18 +138,23 @@
         },
         watch: {
             fullName() {
+                this.validate("fullName")
```

Comment:
Use constant instead of `"fullName"`

Reply:
Validations must be configurable from main component. Think about a usage like this.
```html
<vue-credit-card validator-enabled="true" validation-fields="{cardHolder: true, cardNumber: true, month: true, year: false, cvv: true}" />
```
All the fields except expiry year now must be validated.